Add the integration test script

rustberry opened the pull request:
Currently in the `sendCommand` [section ](https://github.com/omarcostahamido/PVM/blob/99bec6a2aef059e4a5d8c92f75d7cf958f603776/test.py#L15)you are now able to arrange commands to test in one run.

rustberry committed:
Add integration test script

rustberry committed:
Fix program crashing when log directory is not present

KaneBetter commented on test.py:
```diff
@@ -0,0 +1,42 @@
+    print("***err", err)
+    print("***out", out)
+    if "error" in err.lower() or "exception" in err.lower():
+        print("Exceptions or errors detected, test failed.")
```
Maybe we could add a success print after this.

rustberry replied:
Good suggestion!

KaneBetter commented:
Maybe you should add a section in README about how to run our test.

rustberry committed:
Add documentation on tests

rustberry commented:
> Maybe you should add a section in README about how to run our test.

Added a new secondary title in README called `Development` and moved everything related to developing and testing to there (like the `Logs` section).

KaneBetter reviewed (approved):
LGTM

omarcostahamido commented on README.md:
"current directory"? Sounds confusing, could be a little bit more clear. Maybe something like:
```
The PVM execution logs will be recorded in each RPI device memory. These will be stored in the `/log/` directory, which will be created if it doesn't exist yet.
```

Now, when I read this I am assuming that means `PVM/logs/` is that it? Feel free to add more clarification.

rustberry replied:
Here I use "current directory" as a term to describe the working directory of the process/program https://en.wikipedia.org/wiki/Working_directory. But you're right since our reader might not be familiar with it, so I change it to "the working directory of PVM".  If you have any ideas to make it more easier to understand please feel free to comment.

omarcostahamido commented on README.md:
... for each log file...

omarcostahamido commented on README.md:
`Simply run ``python3 test.py`` from the console will run it.` is confusing
maybe `Simply run ``python3 test.py`` on the console to perform the tests.`

omarcostahamido reviewed (commented):
@rustberry 
ok, I see now that it was merged already but I hope you can still address these.
Thank you

edit: hum, good thing that you didn't delete the merged branch?

rustberry commented:
Thanks for the suggestions, made another PR #46 to fix this.

omarcostahamido commented:
ok so then lets get rid of the branches that we know we won't be really needing them anymore, please
@rustberry @KaneBetter @SSSSam141 @xrk-git
thank you
